README.md

```diff
@@ -24,3 +24,14 @@ command and interpreting its output.
 When the plugin detects that macOS is using Dark Mode, it sets the IDEA theme to
 "Darcula". When the plugin detects that macOS is not using Dark Mode, it sets
 the IDEA theme to "IntelliJ".
+
+## How Does It Work on Windows?
+
+The plugin polls the Windows Dark Mode configuration by executing the following
+command and interpreting its output.
+
+    HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Themes\Personalize\AppsUseLightTheme
```

Comment:
> by executing the following command

Is this a command, or a registry key?

Reply:
There is obviously a mistake from my side. One option is to change the phrase quoted by you to "by querying the following registry key" or add "reg query" before the registry key.

Reply:
```suggestion
     reg query HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Themes\Personalize\AppsUseLightTheme
```

Reply:
As stated, this was a problem from my side (classical copy-paste-error). I forgot to change the things.